docs/source/developer/dev.rst

```diff
@@ -0,0 +1,35 @@
+Dev Application
+##############################################################################
+
+This application:
+
+1. Provides a quick reference for seeing what content has been added to the databse.
+2. Allows authors to check markdown content has rendered as expected.
+3. Is to be used in a development environment.
```

Comment:
Might be better if rephrased to 'Is only to be used in a development environment.`